chromium_src/chrome/browser/flags/android/chrome_feature_list.cc

```diff
@@ -23,7 +23,6 @@
 #define kForceWebContentsDarkMode kForceWebContentsDarkMode,            \
     &brave_rewards::features::kBraveRewards,                            \
     &brave_search_conversion::features::kOmniboxBanner,                 \
-    &brave_news::features::kBraveNewsFeature,                           \
     &brave_news::features::kBraveNewsV2Feature,                         \
```

Comment:
LGTM (do we not need to check for the new Brave News V2 feature whether it is enabled or not?)

Reply:
Nope, the Brave News V2 feature is a UI refresh - I'm removing that flag too https://github.com/brave/brave-core/pull/18602